python/pyspark/sql/functions.py

```diff
@@ -44,6 +44,20 @@ def _(col):
     return _
 
 
+def _create_zero_arg_function(name, doc=""):
```

Comment:
`cov()` can provide the covariance between all columns for example. The functions might increase pretty soon!